components/form/builder/src/Range/Default/index.js

```diff
@@ -0,0 +1,132 @@
+import {isValidElement, memo, useCallback, useState, useEffect} from 'react'
+
+import PropTypes from 'prop-types'
+import {field, createComponentMemo} from '../../prop-types/index.js'
+
+import MoleculeInputField from '@s-ui/react-molecule-input-field'
+
+const joinInputValues = inputValues =>
+  inputValues ? Object.values(inputValues).join(',') : ''
+
+const splitInputValues = inputValues => {
+  const [from, to] = inputValues?.split(',') || []
```

Comment:
If you set `[]` as a default param you won't need the `?` and the `|| []` since `[].split(',') = []`

Reply:
Yes, you're right, it's better that way!